src/iuse_actor.cpp

```diff
@@ -248,9 +250,13 @@ long countdown_actor::use( player &p, item &it, bool t, const tripoint &pos ) co
     return 0;
 }
 
-bool countdown_actor::can_use( const player &, const item &it, bool, const tripoint & ) const
+ret_val<bool> countdown_actor::can_use( const player &, const item &it, bool, const tripoint & ) const
 {
-    return !it.active;
+    if( it.active ) {
+        ret_val<bool>::make_failure( "It's already been triggered." );
```

Comment:
This does not have any side effect. Is there a `return` missing?
The message is not translated as well.